edx_proctoring/tests/test_student_view.py

```diff
@@ -365,62 +407,36 @@ def test_get_studentview_unstarted_exam(self):
 
         self._create_unstarted_exam_attempt()
 
-        rendered_response = get_student_view(
-            user_id=self.user_id,
-            course_id=self.course_id,
-            content_id=self.content_id,
-            context={
-                'is_proctored': True,
-                'display_name': self.exam_name,
-                'default_time_limit_mins': 90,
-                'verification_status': 'approved',
-                'verification_url': '/reverify',
-            }
-        )
+        # Verify that the option to skip proctoring is presented
+        rendered_response = self.render_proctored_exam()
         self.assertIn(self.chose_proctored_exam_msg, rendered_response)
         self.assertIn(self.proctored_exam_optout_msg, rendered_response)
 
-        # now make sure content remains the same if
-        # the status transitions to 'download_software_clicked'
+        # Verify that the option to skip proctoring is not presented
```

Comment:
How do you decide when to add a new test vs adding the opposite case inside an existing test?